src/config.py

```diff
@@ -9,7 +9,7 @@
 
 # pylint: disable=invalid-name
 _cfg = None
-cfg: 'Config' = LocalProxy(lambda: _cfg)
+cfg: Config = LocalProxy(lambda: _cfg)
```

Comment:
There was a reason that this was a string -- does this cause any errors?

Reply:
I think putting the typehint in quotes kind of does what PEP 563 does. Previously it failed because the class was declared after the typehint iirc.
Lets see how the ci does.